src/types.ts

```diff
@@ -0,0 +1,28 @@
+import { marginsBottom } from "./structs";
+
+//Accordion
+export interface Data {
+  id: string;
+  titre: string;
+  contenu: string;
+  espacement_bas: keyof typeof marginsBottom;
+}
+
+export interface AccordionProps {
+  data: Data;
+  rows: number;
+}
```

Comment:
```suggestion
type AccordionProps = {
  data: Data;
  rows: number;
}
```
À mettre dans le fichier du composant